### DISCUSSION TITLE
SyntaxError: Unexpected private name #switchChain

### DISCUSSION THREAD

**Discussion by alaaan** [wevm/wagmi#318] [Q&A]
Hi All, 
Im having a bizarre error only in safari... react app runs wonderfully in Chrome.  Does anyone have any suggestions or insights here?  

SyntaxError: Unexpected private name #switchChain. Cannot p
<img width="907" alt="Screen Shot 2022-04-05 at 12 43 33 AM" src="https://user-images.githubusercontent.com/8431313/161680385-85a01d81-3dd5-418d-80fa-923b79b47983.png">
arse class method with private name.

Look to be throwing from ./node_modules/wagmi-core/connectors/walletConnect/dist/wagmi-core-connectors-walletConnect

**Comment by tmm** [comment]
What version of wagmi are you using? This should be fixed in the latest version https://github.com/tmm/wagmi/pull/295
